python-squid/src/main/java/org/sonar/python/DocstringExtractor.java

```diff
@@ -36,14 +37,19 @@
  */
 public class DocstringExtractor {
 
-  public static final Set<AstNodeType> DOCUMENTABLE_NODE_TYPES = ImmutableSet.<AstNodeType>of(
-    PythonGrammar.FILE_INPUT,
-    PythonGrammar.FUNCDEF,
-    PythonGrammar.CLASSDEF);
+  public static final Set<AstNodeType> DOCUMENTABLE_NODE_TYPES = initializeDocumentableNodeTypes();
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Make this member "protected". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS2386)